spec/spec_helper.rb

```diff
@@ -4,6 +4,17 @@ def load_data(path)
   File.read(File.expand_path("../data/#{path}", __FILE__))
 end
 
+def ntp_restrict_value(restrict_entry)
```

Comment:
I think for let you cannot have params, so ideally just keep it as helper, it is fine.